specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json

```diff
@@ -1951,18 +1953,11 @@
             "description": "Restricts the set of items returned."
           },
           {
-            "name": "$top",
+            "name": "$orderby",
             "in": "query",
             "required": false,
-            "type": "integer",
-            "description": "Specifies a non-negative integer n that limits the number of items returned from a collection. The service returns the number of available items up to but not greater than the specified value n."
-          },
-          {
-            "name": "$skip",
-            "in": "query",
-            "required": false,
-            "type": "integer",
-            "description": "Specifies a non-negative integer n that excludes the first n items of the queried collection from the result. The service returns items starting at position n+1."
+            "type": "string",
+            "description": "Specifies the the key by which the result collection should be ordered."
```

Comment:
Same comment as above.

Reply:
It is the former. This API has never supported skip and top. They were added accidentally in the swagger and as part of implementing filter/orderby support they noticed and removed them.